CONTRIBUTING.md

```diff
@@ -0,0 +1,106 @@
+- **Include as much related information** in Issue's description as you can (the predefined template will help you with that)
+  - For bugs: what's happening, how to reproduce it, what OS/version are you using, ...
+- **Tag the Issue with Labels** to categorize it (is it a `bug` or just an `enhancement` like a new feature?). If you're not sure which label to use, leave them empty, someone else will do that (better no labels at all than wrong ones)
+  - `bug` - for broken functionality/UI, bad behaviour
+  - `enhancement/feature` - adding new functionality, tweaks for existing features
+
+It's usually up to Project Manager or someone from the Team to:
+
+- Assign workers to Issues
+- Assign Milestones (AKA "releases") for concrete Issues – please don't assign Milestones to your own Issues if you're not absolutely certain, that the feature/bug _must be_ included in certain release!
+
+---
+
+### 2. Push Requests (PRs)
+
+PRs are working "packages" of code, solving one bug / new feature and are usually connected to existing Issues. For creating new PRs, you must first fork this repository to your own account.
+
+#### 2.1. Create new branch
+
+After your repo is ready, create new branch.
+
+- **Again, keep branch names short and descriptive**
+- Ideally also tag your branch with `prefix/..` – this helps categorizing all the branches at first glance. Some examples:
+  - `fix/..` - for bug fixing, e.g. `fix/createwallet`
+  - `feature/..` - for new features, e.g. `feature/coldstaking`
+  - `gfx/..` - for graphic/UI/UX tweaks, e.g. `gfx/typography`
+
+When your branch is ready, do your changes and push them to the branch. Be sure to push only the changes related to the connected Issue! Solving multiple Issues is also possible, but it's better to focus on one thing at a time.
+
+#### 2.2. Commit your code
+
+- Write meaninful commit messages – don't forget that when your PR is merged, its context is lost. Meaning that once your commits get to `dev` branch, it's harder to say what each commit does just by its name:
+  - ❎ Bad commit message: _"fixed button"_
+  - ✅ Good: _"Receive: fixed 'copy address' button"_
+
+#### 2.3. Submit PR
+
+When your code is complete and ready, [submit new Push Request](https://github.com/particl/partgui/compare).
+
+- **Always set your base branch to `dev`** (never to `master` or others!) – `dev` is our main development branch
+- **As always, write short and descriptive title** (you can keep it the same name as your branch)
+- **Describe what has been done in description** – what does this PR solves? We usually list all the changes and even include a screenshot/GIF overview of the final result. This greatly helps the Reviewers and Testers as they know what to focus on.
+- **Include references to Issues that this PR solves** – Writing `Fixes #[number]` automatically connects this PR to Issue _[number]_ and when this PR gets merged later, the corresponding Issue gets automatically closed.
```

Comment:
writing -> vague. commit message ? PR comment ?

Reply:
Good point. Summarizing these all in PR's description is the best IMO.